internal/cli/atlas/serverless/serverless.go

```diff
@@ -12,18 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package serverlessclusters
+package serverless
 
 import (
 	"github.com/mongodb/mongocli/internal/cli"
 	"github.com/spf13/cobra"
 )
 
 func Builder() *cobra.Command {
-	const use = "serverlessClusters"
+	const use = "serverLess"
 	cmd := &cobra.Command{
 		Use:     use,
-		Aliases: cli.GenerateAliases(use, "sc"),
+		Aliases: cli.GenerateAliases(use, "sl"),
```

Comment:
[nit] `serverless` is one "word" so the camelCase is not correct nor the alias, also what aliases does this generate, it usually goes from plural to singular so I wonder if a word ending with `s` like this is tricking it to do something weird

Reply:
good point, do you think an alias like `server-less` is adequate? Otherwise `serverless` and `sl` will do.

Reply:
I'm ok with just `serverless` I think `sl` a bit of a stretch wordwise but I guess kind of handy for lazy type writers